tests/cypress/e2e/loan/peg-stability-reserves.ts

```diff
@@ -1,5 +1,15 @@
+import { PEG_KEEPERS } from '@/loan/components/PagePegKeepers/constants'
 import { LOAD_TIMEOUT, oneViewport } from '@/support/ui'
 
+/**
+ * Gets pegkeeper cards by test ID suffix.
+ * The complete test ID format is: pegkeeper-card-{contractAddress}-{suffix}
+ * where contractAddress makes each pegkeeper's test IDs unique.
+ */
+function getPegkeeperCards(suffix: string) {
+  return cy.get(`[data-testid^="pegkeeper-card"][data-testid$="-${suffix}"]`)
```

Comment:
I usually create nested testids instead, instead of concatenating - then you don't need to filter out code "in between"